### ISSUE TITLE
Resources not bundled on pypi (0.7 release)

### ISSUE THREAD

**Issue by lordi** [pycollada/pycollada#94]
`resources/schema-1.4.1.xml` is missing when installing via `pip install pycollada`:

```                                                                                                                                         
    def _get(self, path):
>       with open(path, 'rb') as stream:
E       FileNotFoundError: [Errno 2] No such file or directory: '(...)/venv/lib/python3.8/site-packages/collada/resources/schema-1.4.1.xml'

from (...)/collada/schema.py:25
```

I'm using python 3.8.

**Comment by paulmelnikow** [comment]
Hi! I believe this was fixed in #92, though I'm not sure if it's been released yet.

**Comment by jterrace** [comment]
This should be fixed with the 0.7.1 release. Please reopen if there are still any problems.
